README.md

```diff
@@ -227,6 +227,6 @@ Volunteers have translated many of our Contributing, How-to, and Code of Conduct
 You might notice that there are [some missing translations here](docs/README.md#translations) - perhaps you would like to help out by [contributing a translation](docs/CONTRIBUTING.md#help-out-by-contributing-a-translation)?
 
 
-## License
+## License ##
```

Comment:
In GitHub-flavored Markdown (GFM), there is no practical difference between using ## License ## and ## License when formatting headers. Both will be rendered as the same level-two (H2) header with the text "License."